### PR TITLE
Feature/metabase

### PR DESCRIPTION
**Pull request by jwilliams-ocient**
# Overview
This PR contains the Ocient's metabase driver implementation as well as the test extensions necessary to run the Metabase test suite against the Ocient driver. This allows Metabase to leverage Ocient's JDBC driver.

Please read the the [Development Guide](https://www.metabase.com/docs/latest/developers-guide/drivers/start.html) for details w.r.t. project layout.

# Test
Build Ocient driver from source
```
# cd to the metabase root
cd <metabase_root>

# Build Ocient driver
bin/build-driver.sh ocient

# Install the Ocient driver
cp /ocient/db/metabase/resources/modules/ocient.metabase-driver.jar plugins/

# Run the test suite
# NOTE the test suite expects an Ocient cluster to be accessible at tableau-sim.corp.ocient.com:7050. This can be changed
DRIVERS=ocient clojure -X:dev:test
```

### CODE REVIEW

File: test/metabase/test/data/ocient.clj [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+               :dest-table       {:name   (.getString rs fks-table-dest-table-name)
+                                  :schema session-schema}
+               :dest-column-name id-column-key}))))))))
+
+;; overriding describe fks to see if this will allow FKs to work in MB. May not benecessary to enable this to get manual FK support.
+(defmethod driver/describe-table-fks :ocient
+  [_ db-or-id-or-spec-or-conn table & _]
+  ;; Return an empty sequence for the FKs tables themselves
+  ;; (when (not (str/ends-with? db-name-or-nil fks-table-name-suffix))
+  (when true
```
Comment (zzpxyx) [inline test/metabase/test/data/ocient.clj]: Any specific reason to keep this always-true check?

**Review by zzpxyx — APPROVED** [review]
LGTM. Just one minor comment/question below.

File: test/metabase/test/data/ocient.clj (hunk 1 above)
Comment (jwilliams-ocient) [reply test/metabase/test/data/ocient.clj]: Ah, yes, this should be replaced with the line that is commented out above. @Mryan918 can you address this in the PR you open against the `feature/metabase` branch?

Comment (zzpxyx) [reply test/metabase/test/data/ocient.clj]: OK, cool, that makes more sense now. Thanks!

Comment (jwilliams-ocient) [reply test/metabase/test/data/ocient.clj]: Merged, thanks @Mryan918 !

**Pull request merged**